packages/odyssey-react-mui/src/i18n.ts

```diff
@@ -43,14 +43,10 @@ import { translation as vi } from "./properties/ts/odyssey-react-mui_vi";
 import { translation as zhCN } from "./properties/ts/odyssey-react-mui_zh_CN";
 import { translation as znTW } from "./properties/ts/odyssey-react-mui_zh_TW";
 
-import {
-  SupportedLanguages,
-  supportedLanguages,
-} from "./OdysseyTranslationProvider.types";
-
-export const defaultLNG: SupportedLanguages = "en";
+export const defaultLNG = "en";
 export const defaultNS = "translations";
-export const resources = {
+export type OktaI18nResources = Record<string, typeof en>;
+export const resources: OktaI18nResources = {
```

Comment:
this allows us to detect when `ok-PL` and `ok-SK` files are out of sync with the rest. : )

Reply:
> this allows us to detect when ok-PL and ok-SK files are out of sync with the rest. : )

I am curious what this experience looks like for a dev ?

Reply:
<img width="799" alt="Screenshot 2023-12-05 at 9 55 56 AM" src="https://github.com/okta/odyssey/assets/83596595/d5444fe9-ceb5-4d9c-bc6e-b1daddd193ab">

There will be a typescript error :)